CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.14.0)
 project(fortran_stdlib
-        LANGUAGES Fortran
+        LANGUAGES Fortran C CXX
```

Comment:
Don't add languages used for testing only here. Enable them for the compilation of the tests separately instead.

Reply:
Done. Thank you.